webpack.config.babel.js

```diff
@@ -57,6 +57,7 @@ const sassOptions = {
 const config = {
   entry: {
     docs: './packages/bpk-docs/src/index.js',
+    bagCheck: './packages/bpk-docs/src/bag-check-bookmarklet/index.js',
```

Comment:
My only issue with this is that this puts us down a path where we can't use tools like `create-react-app` further down the line...

Reply:
Why aren't we using a raw-loader solution as we did in 41b7cf5bf?

Reply:
I've explained in more details the PR description, but basically because with that we can't update the page that downloads the bookmarklet without having to ask everyone to delete and re add it. This approach will make changing things like changing it to open source transparent to users.

Reply:
As for `create-react-app` we can't do it right now, even without this. The custom static site generator we use is not supported.

Reply:
> As for create-react-app we can't do it right now, even without this. The custom static site generator we use is not supported.

We can - check this out -> https://facebook.github.io/create-react-app/docs/pre-rendering-into-static-html-files